spec/neat/pad_spec.rb

```diff
@@ -25,7 +25,29 @@
 
   context "with argument (default)" do
     it "uses default gutter percentage" do
-      expect(".pad-shorthand-default").to have_rule("padding: 30px 2.35765% 10px 2.35765%")
+      expect(".pad-shorthand-default").to have_rule("padding: 30px 2.35765% 10px")
+    end
+  end
+
+  context "with argument (null)" do
+    it "ignores rules with null values" do
+      ruleset = "padding-right: 10px; " +
+                "padding-left: 10px;"
+      bad_rule = "padding-top: null;"
+
+      expect(".pad-null-values").to have_ruleset(ruleset)
+      expect(".pad-null-values").to_not have_rule(bad_rule)
+    end
+  end
+
+  context "with argument (null default)" do
+    it "sets right and left paddings to gutter percentage" do
+      ruleset = "padding-right: 2.35765%; " +
+                "padding-left: 2.35765%;"
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.